cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1027,7 +1027,9 @@ def make_resource(self, data, **kwargs):
 class _ComputeResourceSchema(BaseSchema):
     """Represent the schema of the ComputeResource."""
 
-    name = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
+    name = fields.Str(
```

Comment:
name validation is already done at config level in `NameValidator`, please remove this

Reply:
Removed